src/material/form-field/testing/form-field-harness-filters.ts

```diff
@@ -14,4 +14,6 @@ export interface FormFieldHarnessFilters extends BaseHarnessFilters {
   floatingLabelText?: string | RegExp;
   /** Filters based on whether the form field has error messages. */
   hasErrors?: boolean;
+  /** Filters based on whether the form field value is valid. */
+  valid?: boolean;
```

Comment:
Should this be called `isValid`? That's the pattern we have for the other filters like `hasErrors`.

Follow-up question: doesn't this overlap with `hasErrors`?

Reply:
Good point about the naming, I'll change it to `isValid`. And, yes there is some overlap but you can have an invalid form-field with no error messages.